libs/sdk-ui-filters/src/DateFilter/DateFilter.tsx

```diff
@@ -123,10 +124,28 @@ export class DateFilter extends React.PureComponent<IDateFilterProps, IDateFilte
         };
     };
 
+    private static checkInitialFilterOption = (filterOption: DateFilterOption) => {
+        if (isAbsoluteDateFilterForm(filterOption) && !(filterOption.from && filterOption.to)) {
+            // eslint-disable-next-line no-console
+            console.warn(
+                "The default filter option is not valid. Values 'from' and 'to' from absoluteForm filter option must be specified.",
+            );
+        }
+
+        if (isRelativeDateFilterForm(filterOption) && !(filterOption.from && filterOption.to)) {
+            // eslint-disable-next-line no-console
+            console.warn(
+                "The default filter option is not valid. Values 'from' and 'to' from relativeForm filter option must be specified.",
+            );
+        }
+    };
+
     constructor(props: IDateFilterProps) {
         super(props);
         this.state = DateFilter.getStateFromProps(props);
 
+        DateFilter.checkInitialFilterOption(props.selectedFilterOption);
```

Comment:
Please put this in `componentDidMount` as that is the place for side-effects, not constructor

Reply:
Function call moved to `componentDidMount`